src/keybindings.c

```diff
@@ -1827,11 +1827,11 @@ static void goto_matching_brace(GeanyDocument *doc)
 
 static gboolean cb_func_clipboard_action(guint key_id)
 {
-	GeanyDocument *doc = document_get_current();
+	GtkWidget *focusw = gtk_window_get_focus(GTK_WINDOW(main_widgets.window));
 
-	if (doc == NULL)
+	if (!IS_SCINTILLA(focusw)) {
```

Comment:
I didn't test, but that seem to be a problem, as `on_cut1_activate()` and friends are supposed to also work on non-Scintilla text widgets (to allow our shortcut to work everywhere).

But the idea of using the focus widget for `COPYLINE` and `CUTLINE` seem sensible.